BDArmory/Targeting/TargetSignatureData.cs

```diff
@@ -177,10 +177,10 @@ public Vector3 predictedPositionWithChaffFactor(float chaffEffectivity = 1f)
                     float distortionFactor = decoyFactor * UnityEngine.Random.Range(16f, 256f);
 
                     // Convert Float jammingFactor position bias and signatureFactor scaling to Vector3 position
-                    Vector3 signatureDistortion = distortionFactor * (vessel.GetSrfVelocity().normalized * -1f * jammingFactor + UnityEngine.Random.insideUnitSphere);
+                    Vector3 signatureDistortion = distortionFactor * (vessel.Velocity().normalized * -1f * jammingFactor + UnityEngine.Random.insideUnitSphere);
 
                     // Higher speed -> missile decoyed further "behind" where the chaff drops (also means that chaff is least effective for head-on engagements)
-                    posDistortion = (vessel.GetSrfVelocity() * -1f * Mathf.Clamp(decoyFactor * decoyFactor, 0f, 0.5f)) + signatureDistortion;
+                    posDistortion = (vessel.Velocity() * -1f * Mathf.Clamp(decoyFactor * decoyFactor, 0f, 0.5f)) + signatureDistortion;
```

Comment:
Similarly, this would be better as
```csharp
posDistortion = signatureDistortion - Mathf.Clamp(decoyFactor * decoyFactor, 0f, 0.5f) * vessel.Velocity();
```